portal/static/js/main.js

```diff
@@ -1894,30 +1894,37 @@ var assembleContent = {
                 };
             };
 
-            var studyId = $("#profileStudyId").val();
-            var siteId = $("#profileSiteId").val();
+            var studyIdField = $("#profileStudyId");
+            var siteIdField = $("#profileSiteId");
+            var hasStudyId = studyIdField.length > 0 && studyIdField.is(":visible");
```

Comment:
Is the visibility of these fields triggered by something else you should be checking the state of instead?

Reply:
@ivan-c ivan, the fields are hidden by css in the stylesheet (e.g. study id is only visible in eproms, site id is only visible in truenthUSA) - that is why I am checking the visibility of each before updating it.